src/main/java/pl/envelo/moovelo/service/event/EventService.java

```diff
@@ -91,6 +97,18 @@ private boolean checkIfEntityExist(Event event) {
         return eventRepository.findById(event.getId()).isPresent();
     }
 
+    public Page<Comment> getAllCommentsByEvent(Event event, CommentPage commentPage){
+        log.info("EventService - getAllCommentsByEvent()");
+        Page<Comment> comments = commentService.getAllCommentsByEvent(event, commentPage);
+
+        if (comments.isEmpty()) {
```

Comment:
myślę, że nie musimy tutaj rzucać wyjątku - brak komentarzy do wydarzenia, to w sumie nie jest wyjątkowa sytuacja